### PR TITLE
Savings tooltip and column

### PR DESCRIPTION
**Pull request by danjm**
This PR adds the final pieces of the latest redesign of the view quote screen: the savings tooltip and modifications the the 3rd column of the select quote popover. This compares to https://github.com/MetaMask/metamask-extension/pull/9629

### CODE REVIEW

File: ui/app/pages/swaps/fee-card/index.scss [hunk 1]
```diff
@@ -228,9 +246,68 @@
...
   }
+
+  &__savings-tooltip {
+    @include H7;
+
+    color: $Grey-500;
+  }
+
+  &__savings-tooltip-card {
+    width: 294.95068359375px;
```
Comment (brad-decker) [inline ui/app/pages/swaps/fee-card/index.scss:257]: this is making the div *bigger* than its allowed screen size... I also don't think it's strictly necessary to make it look good. not a blocker, but these oddly specific pixel values are prone to breaking later.

**Review by brad-decker — CHANGES_REQUESTED** [review]

**Comment by jakehaugen** [comment]
Looking great. Gal and I discussed a few small tweaks:

- Remove verbal explanation of savings. 
- Adjust formatting of line item view (no parent container with border, HR on bottom and top of "Estimated savings" line)
- Add clarification of calculation below savings calculation (i.e. "Savings are calculated by comparing the best quote to the median quote.")
- Change negative values to UI4 (dark gray) vs. red.

Screen Shot 2020-10-23 at 10 29 08 AM

**Comment by jakehaugen** [comment]
We'll also need to cover the scenario when there are no savings. If there are no savings:
- Replace the "Saving ..." text and piggy bank icon with "Using the best quote".
- Do not show the savings tooltip.
- In the view all quotes table, do not show the "Overall savings" column, just highlight the best quote with the "Best" tag. 

Screen Shot 2020-10-23 at 11 13 28 AM

**Comment by danjm** [comment]
The updates to the tooltip designs most recently been requested by @jakehaugen have been added to this PR:
Screenshot from 2020-10-26 11-14-06

**Comment by danjm** [comment]
The update to show the "Using best quote message" when there are no savings has been added to https://github.com/MetaMask/metamask-extension/pull/9629

**Comment by brad-decker** [comment]
@danjm can you rebase this and i'll do another QA/review round

**Comment by danjm** [comment]
Closing in favour of https://github.com/MetaMask/metamask-extension/pull/9905

**Pull request closed without merging**